build_badtrack.py

```diff
@@ -30,6 +30,7 @@
 control_content = """\
 Package: badtrack
 Version: 1.0.0
+Depends: python3
```

Comment:
How do you define multiple dependencies?

Reply:
https://www.debian.org/doc/debian-policy/ch-relationships.html (it's just comma separated)
I'll just add it as a comment so we don't need to look for it later.